Assets/MRTK/SDK/Experimental/Elastic/Scripts/ElasticTypes.cs

```diff
@@ -42,7 +39,77 @@ public struct ElasticExtentProperties<T>
         /// Points inside the extent to which the system will snap.
         /// </value>
         [SerializeField]
-        public T[] SnapPoints;
+        public float[] SnapPoints;
+
+        /// <value>
+        /// Distance at which snap points begin forcing the spring.
+        /// </value>
+        [SerializeField]
+        public float SnapRadius;
```

Comment:
Is it safe to assume that SnapRadius should generally be equal to half the distance between the closest two snap points?

i.e. if the snap radius is actually equal to or larger I assume it ends up being really wonky.

Reply:
Generally yes; however the snap radius can always be significantly _less_ than half the distance between the closest two snap points.

In addition, snap points are not always the "regular, repeating interval" kind of snap points; they can be arbitrarily placed/spaced out, so "half the distance between the closest two snap points" doesn't really have a definite meaning/value.